.github/workflows/ci.yml

```diff
@@ -76,13 +76,8 @@ jobs:
           8.0.x
         source-url: https://nuget.pkg.github.com/open-feature/index.json
 
-    - name: Copy Gherkin
```

Comment:
@Kavindu-Dodan @bacherfl  this is now done as part of the build process itself, [here](https://github.com/open-feature/dotnet-sdk-contrib/pull/189/files#r1583644194)